db/views/rejection_percentage_by_prison_and_calendar_weeks_v02.sql

```diff
@@ -0,0 +1,65 @@
+SELECT rejected.prison_name,
```

Comment:
Have you tried using CTE's? Would help to break the query up to make them more readable